code/datums/dog_fashion.dm

```diff
@@ -179,3 +178,7 @@
 	name = "Festive REAL_NAME"
 	desc = "Ready to party!"
 	obj_icon_state = "festive"
+
+/datum/dog_fashion/head/bishopmitre
```

Comment:
wat

Reply:
That's from an old attempt at getting Ian to wear the bishop hat. I have no clue how that ended up in here but good catch.